helm/prometheus-meta-operator/templates/deployment.yaml

```diff
@@ -46,6 +46,21 @@ spec:
       securityContext:
         runAsUser: {{ .Values.pod.user.id }}
         runAsGroup: {{ .Values.pod.group.id }}
+        runAsNonRoot: true
+      initContainers:
+      - name: volume-mount-hack
+        image: busybox
+        command: ["sh", "-c", "chmod -R 644 /etcd-client-certs"]
```

Comment:
Do we really need this? Can't we set folder/file permissions somewhere else?

Reply:
I do not see any other solution as we are using hostpath volumes

Reply:
Can we discuss about it during refinement ? That looks like an interesting permission topic